packages/theming/android-theme/src/androidTheme.ts

```diff
@@ -4,6 +4,7 @@ import { paletteFromAndroidColors } from './colorsTokens';
 import { androidTypography } from './androidTypography';
 import { getAndroidPalette } from './colorsBase';
 
+// VSO# - Remaning tokens for android.
```

Comment:
If we go have an issue to file, we try to use public GitHub issues with GH#

Reply:
Sounds good, will log a GH issue.